src/CookiePlugin.php

```diff
@@ -109,11 +110,11 @@ private function createCookie(RequestInterface $request, $setCookie)
 
             switch (strtolower($key)) {
                 case 'expires':
-                    $expires = \DateTime::createFromFormat(DATE_COOKIE, $value);
+                    $expires = \DateTime::createFromFormat(\DateTime::COOKIE, $value);
```

Comment:
if i am not mistaken, HttpException is to express a HTTP error code. so it would be TransferException and we say that the server sent unexpected data. 

looking at DateTime::COOKIE having "l, d-M-Y H:i:s T" and Y meaning 4 digit year, i have no idea what hhvm is doing here. but if it works for all versions with a two digit year lets go with that - its just mock data after all.

and lets add a test that triggers the exception, with a date value of "this is not a date" or something.